pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  nodejs,
+  pnpm_9,
+  # cpp is required for building workerd.
+  # workerd is used for some wrangler subcommands.
+  # non-linux platforms may experience errors
+  # on certain subcommands due to this issue.
+  autoPatchelfHook,
+  llvmPackages,
+  musl,
+  xorg,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wrangler";
+  version = "3.62.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "workers-sdk";
+    rev = "wrangler@${finalAttrs.version}";
+    hash = "sha256-/4iIkvSn85fkRggmIha2kRlW0MEwvzy0ZAmIb8+LpZQ=";
+  };
+
+  buildInputs =
+    # cpp is required for building workerd.
+    # workerd is used for some wrangler subcommands.
+    # non-linux platforms may experience errors
+    # on certain subcommands due to this issue.
+    lib.optional stdenv.isLinux
+    [
+      autoPatchelfHook
+      llvmPackages.libcxx
+      llvmPackages.libunwind
+      musl
+      xorg.libX11
+    ]
+
+  nativeBuildInputs =
+    [
+      makeWrapper
+      nodejs
+      pnpm_9.configHook
+    ]
+    ++
+    # cpp is required for building workerd.
+    # workerd is used for some wrangler subcommands.
+    # non-linux platforms may experience errors
+    # on certain subcommands due to this issue.
+    lib.optional stdenv.isLinux autoPatchelfHook;
```

Comment:
Same here regarding the conditional - not needed if `meta.platforms is set correctly.

Reply:
> Same here regarding the conditional - not needed if `meta.platforms is set correctly.

Unresolved.